mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -9,7 +9,7 @@ use common::{StringError};
 use common::wio::{slurp_req};
 use common::executor::{spawn, Timer};
 use common::custom_futures::{join_all_sequential, select_ok_sequential};
-use common::jsonrpc_client::{JsonRpcClient, JsonRpcResponseFut, JsonRpcRequest, JsonRpcResponse, RpcRes};
+use common::jsonrpc_client::{JsonRpcResponseFut, RpcRes, JsonRpcClient, JsonRpcRequest, JsonRpcResponse, JsonRpcRemoteAddr};
```

Comment:
Please reorder imports alphabetically, also feel free to reorder the imports whenever you see that they are not ordered properly.

Reply:
Done